connext_cmake_module/CMakeLists.txt

```diff
@@ -1,9 +1,21 @@
 cmake_minimum_required(VERSION 2.8.3)
 
-project(connext_cmake_module NONE)
+project(connext_cmake_module)
```

Comment:
If you put `NONE` there then this variable `CMAKE_SIZEOF_VOID_P` is not set and you cannot include the `FindConnext.cmake` file in order to fill out the env hook.